altair/widget/js/index.js

```diff
@@ -0,0 +1,70 @@
+import embed from "https://cdn.jsdelivr.net/npm/vega-embed@6/+esm";
+import { debounce } from "https://cdn.jsdelivr.net/npm/lodash-es@4.17.21/lodash.js"
```

Comment:
Know it’s not bundled, but lodash-es is a pretty large dep (98kb minified, https://bundlephobia.com/package/lodash-es@4.17.21 ) for just one import. 

The “modern” lodash alternative I’ve been using are just

Reply:
Awesome, thanks for the recommendation! Switched in [ff29ea1](https://github.com/altair-viz/altair/pull/3119/commits/ff29ea1ef5af054476a1b6320f9cdb1629a390dc)